src/main/groovy/io/openliberty/tools/gradle/utils/LooseEarApplication.groovy

```diff
@@ -64,7 +64,7 @@ public class LooseEarApplication extends LooseApplication {
                     config.addFile(moduleArchive, f, "/WEB-INF/lib/" + f.getName());
                     break
                 case "MF":
-                    addManifestFileWithParent(moduleArchive, f)
+                    addManifestFileWithParent(moduleArchive, f, proj.sourceSets.main.getOutput().getResourcesDir().getParentFile().getCanonicalPath())
```

Comment:
How do we know this is the right path? A comment might help understand what path this is getting and why.